drivers/hmis/app/graphql/mutations/ac_hmis/create_outgoing_referral_posting.rb

```diff
@@ -5,6 +5,7 @@
 ###
 
 module Mutations
+  # Deprecated, remove in release-118
```

Comment:
maybe add todo or die?
```suggestion
   TodoOrDie('Remove deprecated referral mutation', by: '2024-6-15')
```